src/app/onion/relevancy-builder/relevancy-builder.component.ts

```diff
@@ -0,0 +1,322 @@
+    query(
+      ':enter .column',
+      [
+        stagger('150ms ease', [
+          style({ transform: 'translateY(-200px)', opacity: 0 }),
+          animate(
+            '300ms ease',
+            style({ transform: 'translateY(0px)', opacity: 1 })
+          )
+        ])
+      ],
+      { optional: true }
+    ),
+    query('@outcome', [animateChild()], { optional: true })
+  ])
+]);
+
+// This component sets its own page title within the builder store
+@Component({
+  selector: 'clark-relevancy-builder',
+  templateUrl: './relevancy-builder.component.html',
+  styleUrls: ['./relevancy-builder.component.scss'],
+  animations: [
+    trigger('serviceInteraction', [
+      state('open', style({ opacity: '1', transform: 'translateY(-20px)' })),
+      state('closed', style({ opacity: '0', transform: 'translateY(0px)' })),
+      transition('* => *', animate('300ms ease'))
+    ]),
+    builderTransitions
+  ],
+  // these are provided here so that they'll be destroyed when navigating away
+  providers: [BuilderStore, LearningObjectValidator, LearningOutcomeValidator]
+})
+export class RelevancyBuilderComponent implements OnInit, OnDestroy {
+
+  // fires when the component is destroyed
+  destroyed$: Subject<void> = new Subject();
+
+  serviceInteraction: boolean;
+  showServiceInteraction: boolean;
+  removeServiceIndicator: NodeJS.Timer;
+
+  errorMessage: string;
+
+  adminMode: boolean;
+  isRevision: boolean;
+
+  showServiceFailureModal = false;
+
+  historySnapshot: HistorySnapshot;
+
+  // tslint:disable-next-line:max-line-length
+  constructor(
+    private store: BuilderStore,
+    private route: ActivatedRoute,
+    private router: Router,
+    private nav: NavbarService,
+    private builderStore: BuilderStore,
+    private validator: LearningObjectValidator,
+    public noteService: ToastrOvenService,
+    private authService: AuthService,
+    private learningObjectService: LearningObjectService,
+    private history: HistoryService
+  ) { }
+
+  ngOnInit() {
+    this.historySnapshot = this.history.snapshot();
+
+    // listen for route change and grab name parameter if it's there
+    this.route.paramMap
+      .pipe(takeUntil(this.destroyed$))
+      .subscribe(routeParams => {
+        const id = routeParams.get('learningObjectId');
+        const revision = this.route.snapshot.queryParamMap.get('revisionId');
+        const authorUsername = this.route.snapshot.queryParamMap.get('author');
+
+        // if name parameter found, instruct store to fetch full learning object
+        if (revision !== undefined && id) {
+          this.isRevision = true;
+          this.store.isRevision = true;
+          this.store.fetch(id, revision, authorUsername).then(learningObject => {
+            this.setBuilderMode(learningObject);
+          });
+        } else if (id) {
+          this.store.fetch(id).then(learningObject => {
+            if (learningObject.status === LearningObject.Status.RELEASED) {
```

Comment:
All learning objects that are not unreleased can be loaded in here so we should make sure we're not throwing people out of the builder.